### ISSUE TITLE
Old Demos

### ISSUE THREAD

**Issue by DeanGross144** [microsoft/EntraIDGovernance-Training#51]
The links to demos open videos that over 1 year old, are the demos still valid?

**Comment by jorlopama** [comment]
Yes, they are still valid. We are planning on recording new content that may have some updates,  ETA TBD.  If you have one vide that you feel is outdated please let us know through this method (sorry for the long delay)
